src/Game/OptionsManager.php

```diff
@@ -65,6 +66,7 @@ public function __construct()
     {
         $this->options[] = new Option(OptionName::changevote, OptionType::Bool, "on", "When enabled votes can be changed until the final vote is cast.");
         $this->options[] = new Option(OptionName::no_lynch, OptionType::Bool, "on", "When enabled townsfolk can vote not to lynch anybody.");
+        $this->options[] = new Option(OptionName::rolestrategy, OptionType::String, "classic", "Controls how roles are assigned in a new game. Set to classic or vanilla.");
```

Comment:
You asked for constructive criticism on best PHP practices:

http://www.php-fig.org/psr/psr-1/

Class constants like OptionName::rolestrategy should be all uppercase I believe. Whoever implemented !setoption first did not follow this but should be an easy refactor if you choose to do so (not that important)